trinity/rpc/modules/eth.py

```diff
@@ -344,6 +365,24 @@ async def getUncleByBlockNumberAndIndex(self,
         uncle = block.uncles[index]
         return header_to_dict(uncle)
 
+    @format_params(compose(BaseTransactionFields.deserialize, rlp.decode, decode_hex,))
+    async def sendRawTransaction(self,
+                                 transaction: SignedTransactionAPI) -> HexStr:
+
+        validator = get_transaction_validator_for_network_id(
+            self.chain,
+            self.trinity_config.network_id,
+        )
+
+        try:
+            validator.validate(transaction)
+        except ValidationError as err:
+            raise RpcError(err) from err
+
+        await self.event_bus.broadcast(SendLocalTransaction(transaction))
+
+        return encode_hex(transaction.hash)
```

Comment:
```suggestion
        except ValidationError as err:
            raise RpcError(err) from err
        else:
            await self.event_bus.broadcast(SendLocalTransaction(transaction))

            return encode_hex(transaction.hash)
```
If you like, to help visualize that the exception breaks the flow.